README.md

```diff
@@ -0,0 +1,32 @@
+## TDR Jenkinslib
+
+This is a library of groovy functions for the TDR Jenkins set up.
+
+As TDR is a multi-faceted project dealing with many Jenkins builds, a shared library of functions for Jenkins means we can limit the amount of repetitive code. It also allows devs to change code in the Jenkinslib and affect all Jenkins jobs that use that function rather than having to change functions in each Jenkins pipeline.
+
+### Jenkins projects using the Jenkinslib
+
+[E2E tests](https://github.com/nationalarchives/tdr-e2e-tests) should be triggered after:
+* An (integration for now) deployment has successfully built
+* An API deployment has successfully built
+* If the terraform environment changes
+* If there are any changes to the DB.
+
+Jenkins build status posts to Github when:
+* There are changes to the TDR FE project (all branches)
+* When the API has changed
+* When DB has changed
+
+### Set up with Jenkins
```

Comment:
I would reword this section to something like this:

### Jenkins Configuration for Library Functions

TDR Jenkins has been configured to use the library functions with [Docker](https://github.com/nationalarchives/tdr-jenkins)